tests/atomistics/master/test_elastic.py

```diff
@@ -0,0 +1,68 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+import os
+import unittest
+from pyiron.atomistics.structure.atoms import CrystalStructure
+from pyiron_base import Project
+from pyiron.atomistics.master.elastic import calc_elastic_tensor
+import numpy as np
+
+
+class TestMurnaghan(unittest.TestCase):
```

Comment:
Naming error. Shouldn't effect functionality, but definitely confusing for the reader 😉